### PR TITLE
Add 2-year plans constants to lib/plans

### PR DESCRIPTION
**Pull request by adamziel**
This is a first step on the way to implement 2-year plans in Calypso (post with links to relevant discussions: p9jf6J-eR-p2). @dzver implemented three additional products on the backend and this commit reflects that in Calypso. 

Test plan: Run init tests. Go to `/plans` and confirm correct ones are being displayed.

### CODE REVIEW

**Comment by dzver** [comment]
LGTM

**Comment by coderkevin** [comment]
Hi there, the Store tag is for ecommerce functionality on WordPress.com, so I'll remove it here since this is for plans.  👋

File: client/lib/plans/constants.js [hunk 1]
```diff
@@ -182,229 +396,71 @@ export const PLANS_LIST = {
...
-		],
-		getBillingTimeFrame: abtest => {
-			if ( abtest && abtest( 'upgradePricingDisplayV2' ) === 'modified' ) {
-				// Note: Don't make this translatable because it's only visible to English-language users
-				return '/month, billed annually';
-			}
-			return i18n.translate( 'per month, billed yearly' );
-		},
+		getPathSlug: () => 'premium',
+		getPriceTitle: () => i18n.translate( '$99 per year' ), //TODO: DO NOT USE
```
Comment (akirk) [inline client/lib/plans/constants.js]: Please don't make something translatable that is not meant to be translated. It's also a bad idea to hardcode dollar values into a translation. Please either use a variable or not make it translatable at all.

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -182,229 +396,71 @@ export const PLANS_LIST = {
...
+	},
+
+	[ PLAN_PREMIUM_2_YEARS ]: {
+		...getPlanPremiumDetails(),
+		availableFor: plan =>
+			includes( [ PLAN_FREE, PLAN_PERSONAL, PLAN_PERSONAL_2_YEARS, PLAN_PREMIUM ], plan ),
+		getProductId: () => 1023,
+		getStoreSlug: () => PLAN_PREMIUM_2_YEARS,
+		getPathSlug: () => 'premium-2-years',
+		getPriceTitle: () => i18n.translate( '$90 per year' ), //TODO: DO NOT USE
```
Comment (akirk) [inline client/lib/plans/constants.js]: Same here, please don't make something translatable that is not meant to be translated.

File: client/lib/plans/constants.js [hunk 3]
```diff
@@ -182,229 +396,71 @@ export const PLANS_LIST = {
...
-							<strong className="plans__features plan-features__targeted-description-heading" />
-						),
-					},
-				}
-			);
-		},
-		getTagline: () =>
-			i18n.translate(
-				'Learn more about everything included with Business and take advantage of its professional features.'
+		getPriceTitle: () => i18n.translate( '$288 per year' ), //TODO: DO NOT USE
```
Comment (akirk) [inline client/lib/plans/constants.js]: One more instance of translatable text not to be translated (yet?).

File: client/lib/plans/constants.js [hunk 4]
```diff
@@ -182,229 +396,71 @@ export const PLANS_LIST = {
...
-			if ( abtest && abtest( 'upgradePricingDisplayV2' ) === 'modified' ) {
-				// Note: Don't make this translatable because it's only visible to English-language users
-				return '/month, billed annually';
-			}
-			return i18n.translate( 'per month, billed yearly' );
-		},
+		getProductId: () => 1028,
+		getStoreSlug: () => PLAN_BUSINESS_2_YEARS,
+		getPathSlug: () => 'business-2-years',
+		getPriceTitle: () => i18n.translate( '$299 per year' ), //TODO: DO NOT USE
```
Comment (akirk) [inline client/lib/plans/constants.js]: Here too.

**Pull request merged**